extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -691,9 +695,20 @@ export class AmpStoryPage extends AMP.BaseElement {
       if (this.isBotUserAgent_) {
         mediaEl.muted = false;
         mediaEl.removeAttribute('muted');
+        if (mediaEl.tagName === 'AUDIO' && mediaEl.paused) {
+          mediaEl.play();
```

Comment:
why do we want to do this when a bot is crawling the document?

Reply:
+1, was also wondering if this makes a difference

Reply:
I'm trying to be consistent with the rest of the file, where `playAllMedia_` calls play too :)